doc/connectivity/resting_state_networks.rst

```diff
@@ -25,76 +24,66 @@ Multi-subject ICA: CanICA
 
-Data preparation: retrieving example data
------------------------------------------
+Objective
+----------
+ICA is a useful approach for finding independent sources of fMRI
+timeseries covariance without needing to define any priors. ICA
+and similar techniques can be therefore used to define regions or
+networks that share similar BOLD signal across time. The CanICA
+incorporates information both within-subjects and across subjects
+to arrive at consensus components.
 
-We will use down-sampled data from the `brain development dataset
-<https://osf.io/5hju4/files/>`_ has been
-preprocessed using `FMRIPrep and Nilearn <https://osf.io/wjtyq/>`_.
-We use nilearn
-functions to fetch data from Internet and get the filenames (:ref:`more
-on data loading <loading_data>`):
+We can demonstrate this technique using down-sampled data from the 
+`brain development dataset <https://osf.io/5hju4/files/>`_, which
+has been preprocessed using `FMRIPrep and Nilearn <https://osf.io/wjtyq/>`_.
+We use nilearn functions to fetch data from Internet and get the 
+filenames (:ref:`more on data loading <loading_data>`).
 
 
-.. literalinclude:: ../../examples/03_connectivity/plot_canica_analysis.py
-    :start-after: # First we load the brain development fmri data
-    :end-before:  ###############################################
-
-Applying CanICA
----------------
-
+Fitting CanICA model with nilearn
+---------------------------------
 :class:`CanICA` is a ready-to-use object that can be applied to
 multi-subject Nifti data, for instance presented as filenames, and will
 perform a multi-subject ICA decomposition following the CanICA model.
 As with every object in nilearn, we give its parameters at construction,
-and then fit it on the data.
-
-.. literalinclude:: ../../examples/03_connectivity/plot_canica_analysis.py
-    :start-after: # Here we apply CanICA on the data
-    :end-before: ####################################################################
+and then fit it on the data. For examples of this process, see
+here: :ref:`sphx_glr_auto_examples_03_connectivity_plot_compare_decomposition.py`
 
-The components estimated are found as the `components_img_` attribute
-of the object. A 4D Nifti image.
+Once an ICA object has been fit to an fMRI dataset, the individual 
+components can be accessed as a 4D Nifti object using the 
+`components_img_`attribute.
 
 .. note::
-    The `components_img_` attribute is implemented from version 0.4.1 which
-    is easy for visualization without any additional step to unmask to image.
-    For users who have older versions, components image can be done by
-    unmasking attribute `components_`. See :ref:`section Inverse transform:
-    unmasking data <unmasking_step>`.
 
-Visualizing the results
------------------------
+    The use of the attribute `components_img_` from decomposition
+    estimators is implemented from version 0.4.1.
```

Comment:
I think that 0.4.1 has been out for long enough. We don't need to talk about the version when it was added any more.